### PR TITLE
reorder to speed up ui rendering, and bypass wintun uninstall errors

### PR DESCRIPTION
**Pull request by vodkaslime**

### CODE REVIEW

File: node/device/tun/wintun/wintun.go [hunk 1]
```diff
@@ -138,10 +139,12 @@ func (wintun *Adapter) Close() (err error) {
 
 // Uninstall removes the driver from the system if no drivers are currently in use.
 func Uninstall() (err error) {
-	r1, _, e1 := syscall.Syscall(procWintunDeleteDriver.Addr(), 0, 0, 0, 0)
```
Comment (hi-windy) [inline node/device/tun/wintun/wintun.go:141]: I don't think we need to ignore error here and it's better to ignore error at `_ = wintun.Uninstall()`.

File: node/entry/entry_gui.go [hunk 2]
```diff
@@ -112,13 +112,30 @@ func Run() {
...
+
+	zap.L().Info("Startup node successfully")
+
+	if err := app.Run(); err != nil {
+		messagebox.Fatal("Run application failed", err.Error())
+	}
+}
+
+func (app *osApp) Run() error {
 	dev, err := device.NewDevice()
```
Comment (hi-windy) [inline node/entry/entry_gui.go:132]: It's better to move L170 `app.run()` here to show tray first and then initialize the driver.

File: node/device/tun/wintun/wintun.go (hunk 1 above)
Comment (vodkaslime) [reply node/device/tun/wintun/wintun.go:141]: Makes sense

File: node/entry/entry_gui.go (hunk 2 above)
Comment (vodkaslime) [reply node/entry/entry_gui.go:136]: Sure

**Comment by vodkaslime** [comment]
Things start to get more interesting.

1. When I tried to move `app.run()` above, the systray will just not show up at all. Looks like some error happening again. Maybe the `app.run()` is trying to access something, for example device and driver, that will be later initiated?
2. Just as I was about to trouble shoot the issue above, I reran and got a whole new error:
`preflight device failed. error message: Error creating interface: Cannot create a file when that file already exists.`

**Comment by vodkaslime** [comment]
Windows interface and tun creating/uninstall is so messed up. Continuing the investigation.

**Comment by vodkaslime** [comment]
Restarting the app and manually exiting the app solved the 2nd issue, since it helps dispose these stuff. Now focusing on 1st issue

File: node/device/tun/tun_windows.go [hunk 3]
```diff
@@ -85,9 +85,12 @@ func nanotime() int64
...
 	// Remove the previous adapter
-	if err := wintun.Uninstall(); err != nil {
-		return nil, err
-	}
+	// TODO: Uninstall() fails in some machines. So there are indeed errors returned
+	// sometimes in these cases. However in order not to block following steps,
+	// the errors are ignored for now. Deep dive needed.
+	//
+	// tracking issue: https://github.com/pairmesh/pairmesh/issues/65
+	wintun.Uninstall()
```
Comment (hi-windy) [inline node/device/tun/tun_windows.go:93]:
```suggestion
	_ = wintun.Uninstall()
```
Prefer to ignore it explicitly to make others aware it.

**Review by hi-windy — APPROVED** [review]
LGTM

**Pull request merged**